source/features/remove-label-faster.tsx

```diff
@@ -36,7 +36,7 @@ async function removeLabelButtonClickHandler(event: delegate.Event<MouseEvent, H
 }
 
 // TODO: set variable via JSX and inline function in `init` after https://github.com/vadimdemedes/dom-chef/issues/66
-function makeRemoveLabelButton(labelName: string, backgroundColor: string) {
+function makeRemoveLabelButton(labelName: string, backgroundColor: string): HTMLButtonElement {
```

Comment:
Unless we need a specific tag (e.g. to access href), just use `JSX.Element` as the type, so you don’t have to use the assertions below